teuthology/orchestra/console.py

```diff
@@ -271,13 +271,15 @@ def spawn_sol_log(self, dest_path):
         pexpect_templ = \
             "import pexpect; " \
             "pexpect.run('{cmd}', logfile=file('{log}', 'w'), timeout=None)"
-        python_cmd = 'python -c "%s"' % pexpect_templ.format(
-            cmd=console_cmd,
-            log=dest_path,
-        )
+        python_cmd = [
+            '/usr/bin/env', 'python', '-c',
```

Comment:
1) interested in details about spawn termination, but 2) why env here as well?  seems separate  (entirely possible I just don't understand)